templates/react-ts/test/test-context.ts

```diff
@@ -0,0 +1,19 @@
+import { GLOBAL_TEST_RUNNER } from 'socket:test';
+import console from 'socket:console';
+import process from 'socket:process';
+import 'socket:runtime';
+
+// uncomment below to get IPC debug output in stdout
+// import ipc from 'socket:ipc.js'
+// ipc.debug.enabled = true
+// ipc.debug.log = (...args) => console.log(...args)
```

Comment:
@jwerle do we still need these comments?

Reply:
No we shouldn't have this here

Reply:
Comment removed from react-ts template .
I found  the same comment in react template, should I also remove it?

Reply:
There is one in tonic template as well and one in the vanilla template , should I remove those comments  too ?

Reply:
Yeah we should drop them from there too